atd-toolbox/data_model/python/sql_templates/crashes_list_view.sql

```diff
@@ -0,0 +1,148 @@
+        sum(
+            injury_severities.nonincap_injry
+        ) as nonincap_injry_cnt,
+        sum(injury_severities.poss_injry) as poss_injry_cnt,
+        sum(injury_severities.non_injry) as non_injry_cnt,
+        sum(injury_severities.sus_serious_injry) as sus_serious_injry_cnt,
+        sum(injury_severities.fatal_injury) as atd_fatality_cnt,
+        sum(est_comp_cost_crash_based) as est_comp_cost_crash_based
+    from
+        public.crashes as crashes
+    left join public.units as units on crashes.crash_id = units.crash_id
+    left join injury_severities on units.id = injury_severities.unit_id
+    group by
+        crashes.crash_id
+),
+
+geocode_sources as (
+    select
+        cris.crash_id,
+        coalesce(
+            (cris.latitude is null or cris.longitude is null),
+            false
+        ) as has_no_cris_coordinates,
+        case
+            when
+                (edits.latitude is not null or edits.longitude is not null)
+                then 'manual_qa'
+            else 'cris'
+        end as geocode_source
+    from public.crashes_cris as cris
+    left join public.crashes_edits as edits on cris.crash_id = edits.crash_id
+)
+
+select
+    public.crashes.crash_id,
+    public.crashes.case_id,
+    public.crashes.crash_date,
+    public.crashes.address_primary,
+    public.crashes.address_secondary,
+    public.crashes.private_dr_fl,
+    public.crashes.in_austin_full_purpose,
+    public.crashes.location_id,
+    public.crashes.rpt_block_num,
+    public.crashes.rpt_street_pfx,
+    public.crashes.rpt_street_name,
+    public.crashes.rpt_sec_block_num,
+    public.crashes.rpt_sec_street_pfx,
+    public.crashes.rpt_sec_street_name,
+    public.crashes.latitude,
+    public.crashes.longitude,
+    public.crashes.light_cond_id,
+    public.crashes.wthr_cond_id,
+    public.crashes.active_school_zone_fl,
+    public.crashes.schl_bus_fl,
+    public.crashes.at_intrsct_fl,
+    public.crashes.onsys_fl,
+    public.crashes.traffic_cntl_id,
+    public.crashes.road_constr_zone_fl,
+    public.crashes.rr_relat_fl,
+    public.crashes.toll_road_fl,
+    public.crashes.intrsct_relat_id,
+    public.crashes.obj_struck_id,
+    public.crashes.crash_speed_limit,
+    public.crashes.council_district,
+    injury_counts.nonincap_injry_cnt,
+    injury_counts.poss_injry_cnt,
+    injury_counts.sus_serious_injry_cnt,
+    injury_counts.non_injry_cnt,
+    injury_counts.atd_fatality_cnt,
+    injury_counts.unkn_injry_cnt,
+    injury_counts.est_comp_cost_crash_based,
+    lookups.collsn_lkp.label as collsn_desc,
+    geocode_sources.geocode_source,
+    geocode_sources.has_no_cris_coordinates,
+    case
+        when (cris_fatal_injury_counts.cris_fatality_cnt is null) then 0
+        else cris_fatal_injury_counts.cris_fatality_cnt
+    end as cris_fatality_cnt,
+    case
+        when (law_enforcement_fatality_num is not null)
+            then cris_fatality_cnt
+        else 0
+    end as apd_fatality_cnt,
```

Comment:
could you walk me through this logic here? why does `law_enforcement_fatality_num` come into play, and also what is the meaning of `apd_fatality_cnt` according to the vz team and how do they want to use it? also i feel like for clarity `ytd_law_enforcement_fatal_crash` would maybe be a better name? [heres](https://app.gitbook.com/o/-LzDQOVGhTudbKRDGpUA/s/-M4Ve3sp7qA5cPXha0B4/data-dictionary/~/comments?context=post&node=ab6556b2e2ed454684bc6ca1bc53b666) the description of the column

Reply:
ooh, thanks for pointing me to those docs. so for this PR, after i got this basic working view together, i decided to split off this work [into a separate PR](https://github.com/cityofaustin/atd-vz-data/pull/1447). I also wrote up new definitions of all the metrics in the [corresponding issue](https://github.com/cityofaustin/atd-data-tech/issues/17219).

thanks for your suggestion on `ytd_law_enforcement_fatal_crash`. i've been struggling with what to name that column and i really like including `ytd`. so maybe i'll go with `ytd_law_enforcement_fatal_crash_num`?

in any case, i agree the method for `apd_fatality_cnt` is confusing and it's the one i have the most doubts about. let's take that convo to the other PR 👍